package/rmfakecloud-proxy/rmfakecloudctl

```diff
@@ -0,0 +1,319 @@
+#!/usr/bin/env bash
+# Copyright (c) 2021 ddvk
+# Copyright (c) 2021 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+set -euo pipefail
+
+# Path where the certificate authority key and proxy TLS certificate are stored
+data_dir=/opt/var/rmfakecloud-proxy
+
+# Path where system CA certificates are stored
+local_cert_dir=/usr/local/share/ca-certificates
+
+# Path to the static hosts definition
+hosts_path=/etc/hosts
+
+# Path to the configuration directory
+conf_dir=/opt/etc/rmfakecloud-proxy
+
+# Proxy listen address
+proxy_listen=127.0.42.10
+
+# Path to the Xochitl configuration file
+xochitl_conf_path=/home/root/.config/remarkable/xochitl.conf
+
+# Path to Xochitl data
+xochitl_data_dir=/home/root/.local/share/remarkable/xochitl
+
+# Disconnect Xochitl from the cloud
+#
+# This is a necessary step when switching cloud servers to prevent the client
+# from believing all the files have been deleted.
+disconnect-cloud() {
+    if grep -q "^devicetoken=" "$xochitl_conf_path" \
+        && grep -q "^usertoken=" "$xochitl_conf_path"; then
+        echo "Xochitl sync is enabled. Disabling..."
+    else
+        return 0
+    fi
+
+    # Make sure Xochitl is not running
+    local was_active
+
+    if systemctl --quiet is-active xochitl.service 2> /dev/null; then
+        systemctl stop xochitl
+        was_active=1
+    fi
+
+    if pgrep xochitl > /dev/null; then
+        killall xochitl
+        sleep 1s
+    fi
+
+    if pgrep xochitl > /dev/null; then
+        killall -9 xochitl
+        sleep 1s
+    fi
+
+    # Mark all files as not synced
+    grep sync "$xochitl_data_dir"/*.metadata -l \
```

Comment:
Do we ever expect `synced` to not exist in the file?

Reply:
I’m not sure, I took this line from ddvk’s install script

Reply:
I guess if `synced` was missing it would probably be interpreted as `false`, so it's probably fine to leave as is.